exchange/bidder.go

```diff
@@ -115,9 +118,27 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 			bids, moreErrs := bidder.Bidder.MakeBids(request, httpInfo.response)
 			errs = append(errs, moreErrs...)
 			for _, bid := range bids {
+				var targets map[string]string
+				targets = nil
+				if bidderTarg.targetFlag {
+					cpm := bid.Bid.Price
+					width := bid.Bid.W
+					height := bid.Bid.H
+					deal := bid.Bid.DealID
+					cacheKey := ""
+					var err error
+					targets, err = bidder.makePrebidTargets(cpm, width, height, cacheKey, deal, bidderTarg)
+					if err != nil {
+						errs = append(errs, err)
+						// set CPM to 0 if we could not bucket it
+						cpm = 0.0
```

Comment:
Is there any reason to return these bids? I can only imagine it causing support tickets and confusion later.

If not, I think we should just filter these so they never get into the auction.